MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxNetworkRequests.java

```diff
@@ -225,14 +322,29 @@ private boolean executeHttp(URL url, String method, Hashtable<String, String> he
     }
   }
 
-  private String getAuthority(String propertykey) {
+  /**
+   * Calculate the backend URL based on the user environment key variable.
+   *
+   * @param propertykey environment key variable.
+   * @param domain domain to send beacons to (concatenated with propertykey).
+   * @return backend url.
+   */
+  private String getAuthority(String propertykey, String domain) {
     if (Pattern.matches("^[a-z0-9]+$", propertykey)) {
-      return propertykey + ".litix.io";
+      if (domain.startsWith(".")) {
+        return propertykey + domain;
+      } else {
+        return propertykey + "." + domain;
+      }
     }
-    return "img.litix.io";
+    return "img" + domain;
```

Comment:
This needs to _also_ handle the `.` maybe existing or not in this case, if we're not going to change that in Core.

Reply:
This doesn't require a check on the `.` since it's performed in MuxCore.